src/_pytest/nodes.py

```diff
@@ -462,6 +462,7 @@ def reportinfo(self) -> Tuple[Union[py.path.local, str], Optional[int], str]:
     @cached_property
     def location(self) -> Tuple[str, Optional[int], str]:
         location = self.reportinfo()
+        assert isinstance(location[0], py.path.local), location[0]
```

Comment:
You've removed the change from reportinfo, right?  (from the initial Git tree)

Reply:
Yes.